packages/node-opentelemetry/src/otel.ts

```diff
@@ -68,6 +71,17 @@ export const initSDK = (config: SDKConfig) => {
     service: env.OTEL_SERVICE_NAME,
   });
 
+  if (config.advancedNetworkCapture) {
+    if (comparePackageVersions(PKG_VERSION, '==', '0.6.1')) {
+      console.warn(
+        `${LOG_PREFIX} Warning: The "advancedNetworkCapture" flag is available in version 0.6.1 but will be removed in future versions. Please consider using "networkHeadersCapture" and "networkBodyCapture" instead.`,
+      );
+    } else if (comparePackageVersions(PKG_VERSION, '>', '0.6.1')) {
+      throw new Error(
+        `${LOG_PREFIX} The "advancedNetworkCapture" flag is no longer available. Please use "networkHeadersCapture" and "networkBodyCapture" instead.`,
+      );
+    }
+  }
```

Comment:
I think we just need to handle the version > 0.6.1 since 0.6.1 is already published

Reply:
Thanks for the review, changes made